portal-web/test/functional/com/liferay/portalweb/macros/KBArticle.macro

```diff
@@ -864,6 +864,41 @@ definition {
 		AssertVisible(locator1 = "Icon#DELETE");
 	}
 
+	macro selectAssetsToDisplayOnWidget {
```

Comment:
Can you select a folder? If article only please rename to selectArticlesToDisplayOnWidget for clarity

Reply:
Oh nevermind I see https://github.com/liferay-lima/liferay-portal/pull/3795/commits/eff355a316b5f9fc0f8fa3ffbf479f71c02d39cf#diff-69c24ae27b612c28e85d0563fecaecb9cbd92e269ca87140fb38a2bf02604c7fR887-R892 now